src/lib/api-client.js

```diff
@@ -76,14 +94,22 @@ export function searchItem(id) {
 }
 
 export function searchItemForCheckout(address){
-    get('item', {
-        address
-    }).then(data => {
-        Dispatcher.handleAction('CHECKOUT_ITEM_FOUND', {
-            address: data.item.address,
-            status: data.item.status
+    if(!StudentStore.getStudent().hasOverdueItem) {
+        get('item', {
+            address
+        }).then(data => {
+            Dispatcher.handleAction('CHECKOUT_ITEM_FOUND', {
+                address: data.item.address,
+                status: data.item.status
+            });
         });
-    });
+    } else {
+        Dispatcher.handleAction('ERROR', {
+            error:{
+                message: 'Student has at least one overdue item.'
```

Comment:
Based on [the ErrorStore](https://github.com/TheFourFifths/consus-client/blob/feature/check-in-TFF-31-story/src/store/error-store.js#L17), `error` should be the message string. We don't need the `message` part.

E.g.

```javascript
{
    error: 'Student has at least one overdue item.'
}
```